src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -432,23 +428,9 @@
       (or (false? (:authorization-required certificate-status-access-control))
           (not-empty certificate-status-whitelist))
       (log/warn
-        "The 'client-whitelist' and 'authorization-required' settings in the"
-        "'certificate-authority.certificate-status' section are deprecated and"
-        "will be removed in a future release.  Remove these settings and create"
-        "an appropriate authorization rule in the"
-        "/etc/puppetlabs/puppetserver/conf.d/auth.conf file.")
+       (trs "The ''client-whitelist'' and ''authorization-required'' settings in the ''certificate-authority.certificate-status'' section are deprecated and will be removed in a future release.  Remove these settings and create an appropriate authorization rule in the /etc/puppetlabs/puppetserver/conf.d/auth.conf file."))
```

Comment:
Ah ok, I think I was a bit confused about that. I thought that large blocks were to be kept together for the sake of context. In regards to this one it seems like the sentences being taken apart won't really affect the context. I might add some sort of comment for them.